OpenLaTeXStudio/EditorModule/src/latexstudio/editor/EditorTopComponent.java

```diff
@@ -61,20 +69,33 @@
     private File currentFile;
     private DbxState dbxState;
     private String latexPath;
-
+    private Properties OLS_PROPERTIES;
```

Comment:
field names should not be all capital letters; as per Java coding conventions, that's reserved for constants